CHANGELOG.md

```diff
@@ -38,13 +38,17 @@ The category for changes related to documentation, testing and tooling. Also, fo
 
 ### Updates
 
-* Bump Storybook version to include Skip to Content links for keyboard auditors
+* Bump Storybook version to include Skip to Content links for keyboard auditors.
 
     *Katie Foster @inkblotty*
 
+* Update `Button` spacing.
```

Comment:
Can we be more specific about the spacing we're now overriding? Should we provide any feedback when someone tries to set the spacing themselves?